h/static/scripts/annotator/test/integration/anchoring-test.js

```diff
@@ -99,4 +99,24 @@ describe('anchoring', function () {
     ],
     expectFail: true,
   }]);
+
+  unroll('should silently fail when an annotation does not have a quote selector', function (testCase) {
+    var annotation = testCase.annotation;
+    var anchored = guest.anchor(annotation);
+
+    return anchored.then(function (result) {
+      assert.ok(result);
```

Comment:
The `assert.ok()` can be removed since the `deepEqual()` check on the next line will only pass if `result` is defined.